specs/core/0_beacon-chain.md

```diff
@@ -1131,8 +1149,8 @@ def state_transition(state: BeaconState, signed_block: SignedBeaconBlock, valida
 ```python
 def verify_block_signature(state: BeaconState, signed_block: SignedBeaconBlock) -> bool:
     proposer = state.validators[get_beacon_proposer_index(state)]
-    domain = get_domain(state, DOMAIN_BEACON_PROPOSER)
-    return bls_verify(proposer.pubkey, hash_tree_root(signed_block.message), signed_block.signature, domain)
+    message = compute_domain_wrapper_root(signed_block.message, get_domain(state, DOMAIN_BEACON_PROPOSER))
```

Comment:
I know `message` is the name of the param  in the context of `Verify`, but `message` here can easily be conflated with `message` we see in our signed containers (like `signed_block.message`). Maybe we call it `signed_root` or something to disambiguate

Reply:
Want to hear your opinion. I'm no longer sure 😅

Reply:
`signing_root` is a fantastic name that I keep coming back to. 😅

I'm a little reticent about using `signed_root` a signed root is a signature in my mind.

One of the ideas I came up with is `wrapped_root`. In general, I am not 100% satisfied with the names I've implemented and am open to suggestions.

Reply:
I'm actually pretty fine with `message`. Maybe `message_root` but thats a bit long